packages/wiki0/src/server/controllers/authtoken.ts

```diff
@@ -30,7 +30,7 @@ controller.get('/', async (req, res) => {
     grant_type: 'urn:ietf:params:oauth:grant-type:token-exchange',
     requested_token_type: 'urn:ietf:params:oauth:token-type:id-jag',
     subject_token_type: subjectTokenType,
-    resource: `${process.env.TODO_AUTH_SERVER}/token`,
+    resource: `${process.env.TODO_AUTH_SERVER}`,
```

Comment:
nit: We don't need interpolation

```suggestion
    resource: process.env.TODO_AUTH_SERVER,
```